### ISSUE TITLE
[MFR] Stained Pane texture Z fighting

### ISSUE THREAD

**Issue by invinctus** [CoFH/Feedback#101]
I'm using Forge 1.7.10-1291 COFHCore 3.0.0B9-40 and MFR 2.8.0RC6-5.

After placing some stained panes down there seems to be to texture issues where the staining switches between the foreground and background making a strange sort of disco effect.

Here is a screenshot of the issue the pane in the top centre on the left has the staining over the frame and the white bits are overlayed with the red and in the right it is no longer over the frame.
![z-plane](https://cloud.githubusercontent.com/assets/1880751/6125454/24f1a6ca-b10e-11e4-9df9-b9cf2179c1f4.png)

I have tested this with just MFR installed.

**Comment by skyboy** [comment]
Yeah, this is an issue generally persistent in 1.7. Mojang introduced it then fixed it in 1.8.
